lib/health-check.js

```diff
@@ -23,8 +23,11 @@ function callProtect(request, reply) {
   protect(request, reply, _ => reply.send('OK'));
 }
 
-module.exports = function use(fastify, _, done) {
-  fastify.get(readinessURL, { logLevel: 'warn' }, callProtect);
-  fastify.get(livenessURL, { logLevel: 'warn' }, callProtect);
-  done();
+module.exports = function healthCheck(opts) {
+  return (fastify, _, done) => {
+    // Handle health checks
+    fastify.get(opts?.readiness?.path || readinessURL, { logLevel: 'warn' }, opts.readiness || callProtect);
```

Comment:
should these default to warn? or should they be whatever is in the config?

Reply:
@lholmquist done - good catch! Hopefully this PR is good to go now.